src/huggingface_inference_toolkit/diffusers_utils.py

```diff
@@ -47,6 +47,13 @@ def __call__(
         prompt,
         **kwargs,
     ):
+        # diffusers doesn't support seed but rather the generator kwarg
```

Comment:
Maybe add a link to https://github.com/huggingface/api-inference-community/blob/main/docker_images/diffusers/app/pipelines/text_to_image.py#L172-L176 for X-ref/having a source for the future?

Reply:
Ah lol, I just merged and after that this comment popped up. But I'll add it in a separate PR. I think it's a good idea to have to "paper trace"